app/components/WelcomePanel/WelcomeHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import { Heading } from "@nycplanning/streetscape";
+
+export function WelcomeHeader() {
+  return (
+    <Heading
```

Comment:
Is there a reason why this is broken out into its own component and test? I would think if it's just the Heading it can go directly in `WelcomePanel`

Reply:
yes, the desktop and mobile versions of the [WelcomePanel](https://github.com/NYCPlanning/ae-cp-map/blob/876cfc1534abdeac9a7e95ee7d7657f27cf90d50/app/components/WelcomePanel/WelcomePanel.tsx#L43) compose the components differently

Reply:
Is it better to have a separate component or can we assign that JSX to a variable in WelcomePanel and then render it separately that way?

Reply:
Lateral move. Dealer's choice.

Reply:
Fwiw, I think I lean towards making separate components in these scenarios anyway. I know it can feel over-engineer-y but assigning chunks of JSX to one-off variables can get a little messy as applications grow in complexity.